content/tracing/setup/ruby.md

```diff
@@ -18,69 +21,215 @@ further_reading:
 
 ## Installation
 
-To begin tracing applications written in Ruby, first [install and configure the Datadog Agent](/tracing/setup) (see additional documentation for [tracing Docker applications](/tracing/setup/docker/)).
+The following steps will help you quickly start tracing your Ruby application.
+
+#### Setup the Datadog Agent
+
+The Ruby APM tracer sends trace data through the Datadog Agent.
+
+[Install and configure the Datadog Agent](/tracing/setup), see additional documentation for [tracing Docker applications](/tracing/setup/docker/).
+
+#### Quickstart for Rails applications
+
+1. Add the `ddtrace` gem to your Gemfile:
+    
+    ```ruby
+    source 'https://rubygems.org'
+    gem 'ddtrace'
+    ```
+
+2. Install the gem with `bundle install`
+3. Create a `config/initializers/datadog.rb` file containing:
+
+    ```ruby
+    Datadog.configure do |c|
+      # This will activate auto-instrumentation for Rails
+      c.use :rails
+    end
+    ```
+
+#### Quickstart for other Ruby applications
+
+1. Install the gem with `gem install ddtrace`
+2. Include the Ruby application with:
+
+    ```ruby
+    require 'ddtrace'
+    Datadog.configure do |c|
+      # Configure the tracer here.
+      # Activate integrations, change tracer settings, etc...
+      # By default without additional configuration, nothing will be traced.
+    end
+    ```
+
+3. Add manual instrumentation around your code (see "Manual Instrumentation")
+
+#### Final steps
 
-Next, install the ddtrace gem:
+After setting up, you should see traces appear on the [services page](https://app.datadoghq.com/apm/services) within a few minutes. Read more about [using the APM UI](https://docs.datadoghq.com/tracing/visualization/).
```

Comment:
After setting up, your services will appear on the [APM services page](https://app.datadoghq.com/apm/services) within a few minutes. Learn more about [using the APM UI](https://docs.datadoghq.com/tracing/visualization/).